lib/caching-precompiler.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 var path = require('path');
 var fs = require('fs');
-var generateMapFileComment = require('convert-source-map').generateMapFileComment;
```

Comment:
I think it would be cleaner to remove `var convertSourceMap` instead of a shortcut.